torchvision/io/_video_opt.py

```diff
@@ -438,15 +447,20 @@ def _probe_video_from_memory(video_data):
     return info
 
 
-def _convert_to_sec(start_pts, end_pts, pts_unit, time_base):
+def _convert_to_sec(start_pts: float, end_pts: float, pts_unit: str, time_base: Fraction) -> Tuple[float, float, str]:
     if pts_unit == 'pts':
         start_pts = float(start_pts * time_base)
         end_pts = float(end_pts * time_base)
         pts_unit = 'sec'
     return start_pts, end_pts, pts_unit
 
 
-def _read_video(filename, start_pts=0, end_pts=None, pts_unit="pts"):
+def _read_video(
+    filename: str,
+    start_pts: int = 0,
```

Comment:
`start_pts` can be int, float or Fraction, so this should be `Union[float, Fraction]`.

Reply:
Not sure this has been resolved.

Reply:
Oh my bad, I thought it was a bug on github and that was a duplicate for the "end_pnts" comment!